docs/modeling-your-data/modeling-your-data-with-sql-runner/index.md

```diff
@@ -19,8 +19,6 @@ For any new developments we highly recommend using our [dbt packages](/docs/mode
 
 To set up SQL Runner, Snowplow Community Edition users can start from the [User guide](/docs/modeling-your-data/modeling-your-data-with-sql-runner/index.md#user-guide) .
```

Comment:
I would remove this sentence as well... We keep the docs for existing users but should not be nudging new users to set this up